docs/tutorials/clickstream-docker.rst

```diff
@@ -315,55 +317,64 @@ Send the KSQL tables to Elasticsearch and Grafana.
         ksql>
         Exiting KSQL.
 
-2. Navigate to the tutorial directory in the Docker container:
+2. Set up the required Elasticsearch document mapping template
 
    .. code:: bash
 
-       $ cd /usr/share/doc/ksql-clickstream-demo/
+       $ docker-compose exec elasticsearch bash -c '/scripts/elastic-dynamic-template.sh'
```

Comment:
On Ubuntu, this command gives me an error:
```
jim@Maeve:~/repos/ksql/ksql-clickstream-demo$ docker-compose exec elasticsearch bash -c '/scripts/elastic-dynamic-template.sh'

-> Removing kafkaconnect index (if it doesn't exist, this just throws 404)

{"error":{"root_cause":[{"type":"index_template_missing_exception","reason":"index_template [kafkaconnect] missing"}],"type":"index_template_missing_exception","reason":"index_template [kafkaconnect] missing"},"status":404}

-> Loading Elastic Dynamic Template to ensure _TS fields are used for TimeStamp


{"acknowledged":true}
```

Reply:
that's the expected error that the text mentions (404). Should I reword it to make it clearer?

Reply:
I think it would be helpful if you mention it up front in the topic -- I almost gave up, because it looks like a completely fatal error, and "just throws 404" didn't give me confidence. ^_^

Reply:
Good point. I've updated the scripts to be much cleaner in their output.